.pre-commit-config.yaml

```diff
@@ -32,6 +32,7 @@ repos:
           - pep517==0.10.0
           - toml==0.10.2
           - pip==20.3.4
+          - build
```

Comment:
mypy failed:
`error: Module "build" has no attribute "BuildBackendException"  [attr-defined]`

Reply:
should I pin build like other dependencies?

Reply:
Yes, it's make these tests stable.